composer.json

```diff
@@ -25,12 +25,14 @@
         "symfony/yaml": "^5.4 || ^6.3 || ^7.0"
     },
     "require-dev": {
-        "doctrine/annotations": "^1.8.0 || ^2.0",
+        "doctrine/annotations": "^1.13.1 || ^2.0",
```

Comment:
Not really a significant bump, but the Symfony 5.4 branch and ORM requires this at a minimum, so with older Symfony versions not supported an older annotations version isn't really possible.